sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Get-SPOM365AgentAccessInsightsReport.md

```diff
@@ -0,0 +1,181 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-SPOM365AgentAccessInsightsReport 
+applicable: SharePoint Online
+title: Get-SPOM365AgentAccessInsightsReport
+schema: 2.0.0
+author: gchaudhary
+ms.author: gchaudhary
+ms.reviewer:
+manager: lokeshgoel
+---
+
+# Get-SPOM365AgentAccessInsightsReport
+
+## SYNOPSIS
+
+This cmdlet enables the administrator to check status of all active and available reports when no report ID is present and to view or download a report if report ID is present.
+
+> [!NOTE]
+> The feature associated with this cmdlet will be rolling out soon.
+
+## SYNTAX
+
+```
+Get-SPOM365AgentAccessInsightsReport [-ReportId <Guid>] [-Action <ActionType>]
+ [-Content <M365AgentsOnSites>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+If this cmdlet is executed without any parameters, it displays the status of all active and completed reports with the following properties:
+
+| Property             | Description                                                      |
+|:---------------------|:-----------------------------------------------------------------|
+| Id                   | The unique Id of the report.                                     |
+| CreatedDateTimeInUtc | The date and time in UTC when the report creation was triggered. |
+| Status               | The status of the report.                                        |
+| ReportPeriodInDays   | The report duration in days.                                     |

```

Comment:
remove "with the following properties" and remove the table too -- we dont show the output as this changes and we dont keep changing the documentation.